stix2elevator/cli.py

```diff
@@ -70,7 +70,7 @@ def _get_arg_parser(is_script=True):
 
     parser.add_argument(
         "--incidents",
-        help="Incidents will be included in the conversion.  This argument is deprecated.",
+        help="Incidents will be included in the conversion.  Default for version 2.1 is true.",
```

Comment:
Wondering if we should deprecate/remove this option from the CLI and ElevatorOptions since Incidents are now part of the spec... thoughts?

Reply:
It is still valid if the version is 2.0